### DISCUSSION TITLE
There are loopholes in the Vaultwarden versin 1.28.1

### DISCUSSION THREAD

**Discussion by hzpurewater** [dani-garcia/vaultwarden#3654] [Q&A]
Using vulnerability scanning tools to scan Vaultwarden version 1.28.1, many bugs were found。I hope it can be resolved as soon as possible。
The detailed vulnerabilities are as follows:
![22](https://github.com/dani-garcia/vaultwarden/assets/40298933/dec1fabf-aee2-4bf2-a0d9-fcb48c4fc678)

**Comment by BlackDex** [comment]
I'm not seeing bugs linked to Vaultwarden.
What did you used to scan? And what did you scan?

**Comment by BlackDex** [comment]
Vaultwarden doesn't use nginx it self, so thats not vulnerable via that way. Also, Vaultwarden it self is not vulnerable to traversal.

I also do not see how a hacker would have stole your vault. If, then they would have had your credentials.

**Reply by RuneNyhuus** [reply]
Ok thanks for the answer :)

**Comment by BlackDex** [comment]
Also, if you read the page you linked to correctly, it states that both a `location` without a trailing `/` **and** an alias should be used.

**Comment by hzpurewater** [comment]
Sorry, I made a mistake. I scanned the host and found a vulnerability.

There are no issues with Vaultwarden version 1.28.1.
